lib/fastly_nsq/new_relic.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class FastlyNsq::NewRelic
+  include NewRelic::Agent::Instrumentation::ControllerInstrumentation if defined?(::NewRelic)
+
+  attr_reader :agent
+
+  def initialize(agent = NewRelic::Agent)
+    @agent = agent
+  end
+
+  def enabled?
+    @enabled ||= Object.const_defined?('NewRelic')
```

Comment:
did `if defined?(::NewRelic)` not work here?

Maybe we should 

```ruby
begin
  require 'new_relic'
rescue LoadError
end
```

to avoid timing issues.

I think this would also simplifying testing.  What do you think?

Reply:
`defined?` doesn't return a boolean which is annoying. That `?` is misleading.

Good call. I'll add the `require; rescue;`